packages/manager/apps/hycu/src/utils/tests/setup-test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { SetupServer } from 'msw/node';
+import { i18n } from 'i18next';
+import { I18nextProvider } from 'react-i18next';
+import {
+  ShellContext,
+  ShellContextType,
+  initShellContext,
+} from '@ovh-ux/manager-react-shell-client';
+import { render, waitFor, screen } from '@testing-library/react';
+import {
+  getServicesMocks,
+  GetServicesMocksParams,
+} from '@ovh-ux/manager-react-components/src/hooks/services/mocks/services.mock';
+import { expect } from 'vitest';
+import { toMswHandlers } from '../../../../../../../playwright-helpers';
+import { getAuthenticationMocks } from '../../../../../../../playwright-helpers/mocks/auth';
+import { getLicenseHycuMocks, GetLicenseHycuMocksParams } from '../../mocks';
+import { initTestI18n } from './init.i18n';
+import { TestApp } from './TestApp';
+
+let context: ShellContextType;
+let i18n: i18n;
+
+export const setupTest = async (
+  mockParams: GetLicenseHycuMocksParams & GetServicesMocksParams = {},
+) => {
+  (global.server as SetupServer)?.resetHandlers(
+    ...toMswHandlers([
+      ...getAuthenticationMocks({ isAuthMocked: true }),
+      ...getLicenseHycuMocks(mockParams),
+      ...getServicesMocks(mockParams),
+    ]),
+  );
+
+  if (!context) {
+    context = await initShellContext('hycu');
+  }
+
+  if (!i18n) {
+    i18n = await initTestI18n();
+  }
+
+  const result = render(
+    <I18nextProvider i18n={i18n}>
+      <ShellContext.Provider value={context}>
+        <TestApp />
+      </ShellContext.Provider>
+    </I18nextProvider>,
+  );
+
+  await waitFor(
+    () =>
+      expect(
+        screen.getAllByText('title', { exact: false }).length,
```

Comment:
'title' label will appear on the listing page ?

Reply:
I think change it with bug issue about product name not translated. And implement hycu.config.ts as veeam-backup.

But It's to have test success at this step

Reply:
But May be it's wrong if title is not translated ? Because run success with this param

Reply:
In Veeam Backup some constants are not translated that's why we put them in constant files. If it's the same for HYCU, yes you should use constants too.

But if you use a translated key, it will be translated in the tests I think.